modules/ROOT/pages/footnotes.adoc

```diff
@@ -0,0 +1,50 @@
+= Footnotes plugin
+:navtitle: Footnotes
+:description: Insert footnotes into the {productname} editor
+:keywords: footnotes, footnote, insertfootnotes, insert, footnotesupdate, mceInsertFootnote, mceUpdateFootnotes
+:pluginname: Footnotes
+:plugincode: footnotes
+:pluginminimumplan: tierthree
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.2v.adoc[]
+
+The {pluginname} plugin allows the user to insert a footnote into the {productname} editor.
+
+Invoking the plugin adds a superscripted footnote number at the current insertion point, or at the end of the current selection.
+
+The plugin then places the insertion point in a new line in the footnotes section located at the bottom of the editor, allowing the user to add the footnote entry related to the newly-created footnote.
```

Comment:
Is `insertion point` really clearer than `cursor` here?

Reply:
**cursor**

The UI element that shows where input will be accepted.

At a shell prompt, this has traditionally been a rectangle or underscore glyph, sometimes flashing, sometimes not.

In strings of text in GUI-centric environments (whether the string is being displayed by a text editor or word processor) the cursor is almost always a flashing vertical line.

Many shell prompts have, since the rise of GUIs, also adopted the flashing vertical line as the cursor UI element.

**insertion point**

The place within a document where input will appear.

A cursor can show where the insertion point is, but it is not the insertion point.

It is, absolutely, a fine distinction. But the insertion point is not always shown with a cursor.

**pointer** 

The object on a screen that is controlled by, and acts as the UI proxy for, the mouse (or equivalent pointing device).

Most UI systems change the pointer’s appearance on a contextual basis.

The default arrow pointer is for selecting objects and pressing on-screen buttons.

The I-beam pointer is for selecting text strings and setting the cursor in a text string.

Clicking the I-beam pointer in a text string sets a new insertion point. And the cursor is displayed at this new point to provide visual confirmation of the new insertion point.

And, in the case of the footnotes plugin, the plugin moves the insertion point to a new line in the footnotes section (and the vertical line cursor starts flashing at this new point to show this).